src/Program.fs

```diff
@@ -400,10 +407,29 @@ let generate (configFile: string) =
                 colorprintfn "✏️  Generating Fable project $green[%s]" projectPath
                 let files =
                     generatedFiles
-                    |> Seq.map (fun file -> sprintf "        <Compile Include=\"%s\" />" (Path.GetFileName file))
-                    |> String.concat "\n"
-
-                File.WriteAllText(projectPath, CodeGen.sampleFableProject files)
+                    |> Seq.map
+                        (fun file ->
+                            XElement.ofStringName("Compile",
+                                XAttribute.ofStringName("Include", file |> Path.GetFileName)
```

Comment:
Can you please simplify single function application usage of the `|>` operator?
```fs
XAttribute.ofStringName("Include", file |> Path.GetFileName)
```
becomes
```fs
XAttribute.ofStringName("Include", Path.GetFileName file)
```